### PR TITLE
migrate CIs to GitHub Actions

### PR DESCRIPTION
**Pull request by lufia**
I've migrate both Travis CI and AppVeyor tasks into a GitHub Actions workflow.

see mackerelio/mkr#339

### CODE REVIEW

**Comment by lufia** [comment]
I have checked the workflow generates 9 packages and they will be uploaded at the next release.

ref https://github.com/mackerelio/go-check-plugins/pull/432/checks?check_run_id=1471763258#step:6:920

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -34,9 +40,47 @@ jobs:
...
+          go get github.com/mackerelio/golib/cmd/mackerel-github-release
+          make clean build rpm deb
+          mackerel-github-release && mackerel-github-release -staging
+        env:
+          GITHUB_TOKEN: ${{ secrets.MACKERELBOT_GITHUB_TOKEN }}
+      - uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          fields: repo,message,commit,action,eventName,ref,workflow,job,took
+          username: mkr-release
```
Comment (shibayu36) [inline .github/workflows/test.yml:84]: Is `go-check-plugins-release` correct?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
       - run: |
-          make lint cover all
```
Comment (shibayu36) [inline .github/workflows/test.yml]: `all` task in Makefile may be erased?

**Comment by shibayu36** [comment]
`all` task in Makefile may be erased?

**Review by shibayu36 — APPROVED** [review]
I approved, but please fix slack username.

File: .github/workflows/test.yml (hunk 2 above)
Comment (lufia) [reply .github/workflows/test.yml]: Sure, but I think `make all` is useful to run all tasks locally...

File: .github/workflows/test.yml (hunk 1 above)
Comment (lufia) [reply .github/workflows/test.yml]: fixed.

**Pull request merged**